api/v1alpha1/virtualmachineimage_types.go

```diff
@@ -41,6 +41,14 @@ type VirtualMachineImageOSInfo struct {
 	Type string `json:"type,omitempty"`
 }
 
+type VirtualMachineImageInfo struct {
+	// ImageName describes the display name of this VirtualMachineImage.
+	ImageName string `json:"imageName,omitempty"`
+
+	// ImageSourceID describes the uuid of the content library which owns this VirtualMachineImage.
+	ImageSourceID string `json:"imageSourceId,omitempty"`
```

Comment:
Should we include a ImageSourceName as well (display name of the content library?)